packages/account-web-vault/src/web-user.ts

```diff
@@ -0,0 +1,250 @@
+import { Network, Context, Profile } from '@verida/client-ts';
+import { VaultAccount, hasSession, VaultAccountConfig } from './index';
+import { EventEmitter } from 'events'
+import { MessageSendConfig } from '@verida/client-ts/dist/context/interfaces';
+
+const VeridaEvents = new EventEmitter()
+
+export interface WebUserProfile {
+    name?: string
+    avatarUri: string
+    country?: string
+    description?: string
+}
+
+export interface WebUserConfig {
+    accountConfig: VaultAccountConfig
+    clientConfig: ClientConfig
```

Comment:
The `client` can also be used in situation when there is non account connected.
In such situation, it makes sense to create a `Client` beforehand.

It would be nice to be able to pass that instance instead of re-creating a new one from the `ClientConfig`.

Can it be problematic to have multiple instances of `Client`? If not, then probably not a big deal to re-create one when configuring `WebUser`

Reply:
Good point.

What if “getClient()” always returned a client object?

Reply:
Added `getClient()`

Reply:
It means you have to configure WebUser, but I guess it's ok.

If you use WebUser to get the client, it means you need WebUser for something else, so it make sense to configure it.
If you don't want to configure it, it's because you don't need WebUser, so you should just use `new Client(...)`.